src/internal/components/InternalThemeEditor.tsx

```diff
@@ -21,8 +22,8 @@ type InternalThemeEditorProps = {
   publishThemeConfiguration: (data: any) => void;
   themeConfig?: ThemeConfig;
   saveThemeSaveState: (data: any) => void;
-  themeHistory?: ThemeSaveState;
-  setThemeHistory: (themeHistory: ThemeSaveState) => void;
+  themeHistory?: ThemeHistories;
```

Comment:
Call this themeHistories to be consistent with the type. There is another type called ThemeHistory so then the variable is confusing since it's referring to something different.